apps/worker/src/app/shared/utils/cron-health.ts

```diff
@@ -0,0 +1,14 @@
+import { Logger } from '@nestjs/common';
+
+const url = process.env.CRON_CHECKIN_URL
+  ? process.env.CRON_CHECKIN_URL
+  : 'https://uptime.betterstack.com/api/v1/heartbeat/';
```

Comment:
Are we sending a beat to an external service after each job is completed?
Are we going to individually do that for every queue? If it is to provide uptime stats I think how is it implemented only sends the liveness of the job completed metrics queue but not the actual queues that process jobs. Am I missing anything?

Reply:
Yes, we are sending it to better uptime.

You are correct this is just for the metric queue. These queues are critical for auto-scaling the workers to ensure that there are enough.

Reply:
Oh I get it now. So it is to based on active jobs and completed jobs understand the pressure of the system. 👍🏻

Reply:
Correct, as that is all we need. If processing stopped tomorrow because there was no use then we would not need to alert just check the pressure and scale down.